Automattic/pocket-casts-android
# Fix: OPML Import

## Description
- This PR addresses two issues. The first one is that we were not displaying any message error when we try to import a invalid OPML file and the other issue is that we were not able to import OPML from Google Podcast app. This is because Google was not removing invalid xml characters so we could not parse the file
- See Internal Link: p1705580029359379/1705580020.681289-slack-C02A333D8LQ)

Closes #1716
Closes #2259 


## Testing Instructions
1. Download the file p1705580029359379/1705580020.681289-slack-C02A333D8LQ
2. In your device files, tap on this OPML file and import to Pocket Casts (You can tap on share button and select Pocket Casts app)
3. Go to Podcasts tab
4. ✅ Make sure the podcasts were imported 

## Screenshots or Screencast 

https://github.com/Automattic/pocket-casts-android/assets/42220351/f4320e5e-f225-439c-b224-de76de3dad7a



## Checklist
 
#### I have tested any UI changes...
